website/content/docs/security/security-models/core.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
-page_title: Consul Core Security Model
+page_title: Security Models: Consul Core
```

Comment:
It seems weird to refer to a titled entity called "Consul Core" here and nowhere else.

Maybe this should just be
```suggestion
page_title: Security Models: Consul
```